controllers/kubeobjects.go

```diff
@@ -650,6 +655,7 @@ func backupCreate(
 
 	backupSpec.StorageLocation = backupNamespacedName.Name
 	backupSpec.IncludedNamespaces = []string{sourceNamespaceName}
+	backupSpec.SnapshotVolumes = newFalse()
```

Comment:
I suggest `newFalse()` be replaced with `new(false)`, where new is a special built-in "routine" to create a pointer to a `bool` in this case

Reply:
I used `backupSpec.SnapshotVolumes = new(false)` and it gave this error when doing `make docker-build`:
```sh
# github.com/ramendr/ramen/controllers
controllers/kubeobjects.go:653:35: false is not a type
```
so I changed to write a newFalse() function.

Reply:
Sorry, wrong syntax it seems